src-tauri/src/main.rs

```diff
@@ -210,7 +220,10 @@ async fn main() {
         })
         .on_window_event(move |event| {
             if let WindowEvent::CloseRequested { api, .. } = event.event() {
-                if wooting_macro_backend::config::ApplicationConfig::read_data().minimize_to_tray {
+                if ApplicationConfig::read_data()
+                    .unwrap_or_else(|err| panic!("Error reading config data: {}", err.to_string()))
+                    .minimize_to_tray
```

Comment:
Couldn't this just be a `.expect()`? It should include the context of the error as well right?

Reply:
The text you're adding could also provide a little more context.
e.g. "Error reading config data when determining how to handle close request" or something. 

I suspect the error itself will already tell you that config data reading failed, the additional bit you should add is what you're trying to do with it, then it should help narrow the source of the panic if all you have is the message

Reply:
Tried to tweak here: [90a6117](https://github.com/WootingKb/wooting-macros/pull/194/commits/90a61171b79f195e6d0ede1ebe32a2fe475f546b)